test/helpers/arvm_test_models.rb

```diff
@@ -30,19 +30,24 @@ class ViewModelBase < ViewModel::ActiveRecord
   self.abstract_class = true
 
   module ContextAccessLogging
-    attr_accessor :edit_checks, :visible_checks
+    attr_accessor :editable_checks, :valid_edit_checks, :visible_checks
 
     def initialize(**args)
       super
 
       # force existence of these objects, so when we get cloned in context we
       # get aliased.
-      @edit_checks    = []
-      @visible_checks = []
+      @editable_checks   = []
+      @valid_edit_checks = []
+      @visible_checks    = []
     end
 
-    def log_edit_check(viewmodel)
-      edit_checks << viewmodel.to_reference
+    def log_editable_check(viewmodel)
+      editable_checks << viewmodel.to_reference
```

Comment:
semi difficult: note that it's stashing the call to `editable?` whether or not its result was used.